static/js/API.js

```diff
@@ -77,12 +77,11 @@ export const filterOutEmptyValues = (params, removeEmptyArrays = true) => {
     // Also, the number 0 may be a valid input but evaluate to false,
     // so just let numbers through
     if (
-      (!(
+      !(
         Array.isArray(params[key]) &&
         params[key].length === 0 &&
         removeEmptyArrays
-      ) &&
-        params[key]) ||
```

Comment:
@Theodlz I was super confused, but this line was preventing passing false to a handler?!!?

Reply:
hmmmm that would be super weird! Is that something I added??? There must be a reason why that's here

Reply:
Though it might not be done properly, I think this is important. Because when deleting it you make it possible to send values like null, undefined, or empty strings that might break the API calls. Might want to look for anything BUT false.

Reply:
Ok that would make sense. Right now you literally can't send false, so some exception needs to be made!

Reply:
Ok should work now, I'll let you take a look?